src/lake/Lake/CLI/Init.lean

```diff
@@ -287,6 +303,17 @@ def validatePkgName (pkgName : String) : LogIO PUnit := do
   if pkgName.toLower ∈ ["init", "lean", "lake", "main"] then
     error "reserved package name"
 
+def createLeanActionWorkflow (dir : FilePath) : LogIO PUnit := do
+  logInfo "creating lean-action CI workflow"
+  let workflowDir := dir / ".github" / "workflows"
+  let workflowFile := workflowDir / "lean_action_ci.yml"
+  if (← workflowFile.pathExists) then
+    logWarning "lean-action CI workflow already exists"
```

Comment:
```suggestion
    logVerbose "lean-action CI workflow already exists"
```

`lake init` / `lake new` do not generally perform step-by-step logging. However, adding some is probably not a bad idea. If we do log here, since logging it is not currently expected, I think this (and below) should use `logVerbose`. That way, it is only printed with `-v` (or on an error).

Reply:
Makes sense. Done :+1: